addOns/pscanrulesAlpha/src/test/java/org/zaproxy/zap/extension/pscanrulesAlpha/Base64DisclosureUnitTest.java

```diff
@@ -53,4 +59,41 @@ void shouldReturnExpectedMappings() {
                 tags.get(CommonAlertTag.OWASP_2017_A03_DATA_EXPOSED.getTag()),
                 is(equalTo(CommonAlertTag.OWASP_2017_A03_DATA_EXPOSED.getValue())));
     }
+
+    @ParameterizedTest
+    @CsvSource(
+            value = {
+                "ETag",
+                "etag",
+                "ETAG",
+                "Authorization",
+                "X-ChromeLogger-Data",
+                "X-ChromePhp-Data"
+            })
+    void shouldNotAlertOnIgnoredHeader(String headerName)
+            throws URIException, HttpMalformedHeaderException, NullPointerException {
+        // Given
+        HttpMessage msg = new HttpMessage(new URI("https://example.com/", false));
+        // This value is from Issue 6619, that it's recognized as base64 is all that matters (not
+        // what the content actually is)
+        msg.getResponseHeader().addHeader(headerName, "W/\"45cd-7MlSATHznbvb7qdGT+/VL6oqXM\"");
```

Comment:
Extract a constant and add the comment there, though it does not seem to be needed.

Reply:
I guess I could make a specific test method with etag and this string to prove the issue reported is addressed and then add a constant that's "foobar" or "example" base64d for other tests. That's probably easier/more clear.

Reply:
```js
console.log(btoa('foobar'))
Zm9vYmFy

console.log(btoa('example'))
ZXhhbXBsZQ==
```

Reply:
They can check the revision.